data/tabular/qm9/meta.yaml

```
@@ -0,0 +1,187 @@
+    - The molecule with the {smiles__description} representation of {smiles#} has an energy of highest occupied molecular orbital {homo#} Hartree.
+    - The molecule with the {inchi__description} representation of {inchi#} has an energy of highest occupied molecular orbital {homo#} Hartree.
+    - The molecule with the {smiles__description} representation of {smiles#} has an energy of lowest occupied molecular orbital {lumo#} Hartree.
+    - The molecule with the {inchi__description} representation of {inchi#} has an energy of lowest occupied molecular orbital {lumo#} Hartree.
+    - The molecule with the {smiles__description} representation of {smiles#} has a homo-lumo gap of {gap#} Hartree.
+    - The molecule with the {inchi__description} representation of {inchi#} has a homo-lumo gap of {gap#} Hartree.
+    - The molecule with the {smiles__description} representation of {smiles#} has an electronic spatial extent of {r2#} Bohr^2.
+    - The molecule with the {inchi__description} representation of {inchi#} has an electronic spatial extent of {r2#} Bohr^2.
+    - The molecule with the {smiles__description} representation of {smiles#} has a zero point vibrational energy of {zero_point_energy#} Hartree.
+    - The molecule with the {inchi__description} representation of {inchi#} has a zero point vibrational energy of {zero_point_energy#} Hartree.
+    - The molecule with the {smiles__description} representation of {smiles#} has an internal energy of {u0#} Hartree at 0 K.
+    - The molecule with the {inchi__description} representation of {inchi#} has an internal energy of {u0#} Hartree at 0 K.
+    - The molecule with the {smiles__description} representation of {smiles#} has an internal energy of {u298#} Hartree at at 298.15 K.
+    - The molecule with the {inchi__description} representation of {inchi#} has an internal energy of {u298#} Hartree at at 298.15 K.
+    - The molecule with the {smiles__description} representation of {smiles#} has an enthalpy of {h298#} Hartree at at 298.15 K.
+    - The molecule with the {inchi__description} representation of {inchi#} has an enthalpy of {h298#} Hartree at at 298.15 K.
+    - The molecule with the {smiles__description} representation of {smiles#} has a gibbs free energy of {g298#} Hartree at at 298.15 K.
+    - The molecule with the {inchi__description} representation of {inchi#} has a gibbs free energy of {g298#} Hartree at at 298.15 K.
+    - The molecule with the {smiles__description} representation of {smiles#} has a heat capacity of {heat_capacity#} cal/(mol K) at at 298.15 K.
+    - The molecule with the {inchi__description} representation of {inchi#} has a heat capacity of {heat_capacity#} cal/(mol K) at at 298.15 K.
+    - |-
+      'Question: What is a molecule with a homo-lumo gap of {gap#} Hartree and an energy of highest occupied molecular orbital {lumo#} Hartree?
```

```suggestion
      'Question: What is a molecule with a homo-lumo gap of {gap#} Hartree and an energy of highest occupied molecular orbital {lumo#} Hartree?
```

is `lumo` correct here?

---

Thank you so much for the careful review 🙏.
It should be  '''lumo'''